src/components/app/header/index.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react'
 import {FunctionComponent} from 'react'
 import Link from '../../link'
+import NextLink from 'next/link'
 import Image from 'next/legacy/image'
 import Eggo from '@/components/icons/eggo'
 import {useViewer} from '@/context/viewer-context'
 import {track} from '@/utils/analytics'
 import {isEmpty} from 'lodash'
 import FeedbackInput from '@/components/feedback-input'
-import useBreakpoint from '@/utils/breakpoints'
```

Comment:
I went ahead and removed all the js breakpoint stuff and replaced it with with css which lets us avoid `isMounted` check and hydration mismatches.